Checkout/Source/Network/Models/SecurityCode/SecurityCodeResponse.swift

```diff
@@ -0,0 +1,14 @@
+// 
+//  SecurityCodeResponse.swift
+//  
+//
+//  Created by Okhan Okbay on 05/10/2023.
+//
+
+import Foundation
+
+public struct SecurityCodeResponse: Decodable, Equatable {
+  public let type: String
+  public let token: String
+  public let expiresOn: String
+}
```

Comment:
good shout! 

In addition, we can add some sample responses to denote that type means `cvv` string and can only mean that. Also we can imply the format of the expiry date that's returning here. What do you think? @chintan-soni-cko

Reply:
Yes, it would be nice to have definitely. However, in this case, they will check in-depth details from the Checkout public docs as well (once they publish for CVV) because we covered almost all the details for the parameters in the SDK documentation.

Reply:
Thanks for this. The current comments on Android code seems like repeating what the code is already speaking about without the comments, I believe. I'm not saying they are redundant but they don't feel like crucial. I think we can either add more context or can discuss the value of the comments otherwise @chintan-soni-cko

Reply:
added comments @chintan-soni-cko